sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/PushTokenApiRequest.kt

```diff
@@ -57,6 +57,13 @@ internal class PushTokenApiRequest(
     override val successCodes: IntRange get() = HTTP_ACCEPTED..HTTP_ACCEPTED
 
     override var body: JSONObject? = null
+        set(value) {
+            if (!this::initialBody.isInitialized) {
+                initialBody = value?.toString() ?: ""
```

Comment:
Could you just initialize `initialBody` to empty string you do that by default or would that be confusing?

Reply:
Its a fair point, that would eliminate the risk of using `lateinit` at all.

Reply:
tbh though I don't know much about lateinit and when to use it.

Reply:
Its a way to say "this property is non-nullable, but I can't set it in the constructor... so I promise not to access it before its set" and will crash if you break your promise. Has some inherent risk